docs/applications/big-data/big-data-in-the-linode-cloud-streaming-data-processing-with-apache-storm.md

```diff
@@ -0,0 +1,1095 @@
+
+1.  The scripts communicate with Linode's API using Python. Install Git, Python 2.7 and curl on your Ubuntu 14.04 LTS or Debian 8 workstation or Linode:
+
+        sudo apt-get install python2.7 curl git
+
+2.  Download the project git repository:
+
+        git clone "https://github.com/pathbreak/storm-linode"
+        cd storm-linode
+        git checkout $(git describe $(git rev-list --tags='release*' --max-count=1))
+
+3.  Make the shell and Python scripts executable:
+
+        chmod +x *.sh *.py
+
+4.  Make a working copy of the API environment configuration file:
+
+        cp api_env_example.conf api_env_linode.conf
+
+5.  Open `api_env_linode.conf` in a text editor, and set `LINODE_KEY` to the API key previously created (see [Get a Linode API key](#get-a-linode-api-key)).
+
+    {: .file-excerpt}
+    ~/storm-linode/api_env_linode.conf
+    :   ~~~
+        export LINODE_KEY=Enter your API key here
+        ~~~
+
+    Save and close the editor.
+
+6.  Open `~/storm-linode/cluster_manager.sh` in a text editor and change the following configuration settings to customize where and how the Cluster Manager Linode is created:
+
+    -  `ROOT_PASSWORD`:
+        This is the root user's password for the Cluster Manager Linode and is **required** to create the node. Set this to a secure password of your choice. Linode requires the root password to contain at least 2 of these 4 character types – lower case characters, upper case characters, numeric characters and symbolic characters.
+
+        If you have spaces in your password, make sure the entire password is enclosed in double quotes. If you have double quotes, dollar characters or backslashes in your password, escape each of them with a backslash (`\`).
+
+    -  `PLAN_ID`:
+        The default value of `1` creates the Cluster Manager Linode as a 2GB node, the smallest plan. This is usually sufficient. However, if you want a more powerful Linode, use the following commands to see a list of all available plans and their IDs:
+
+            source ~/storm-linode/api_env_linode.conf
+            ~/storm-linode/linode_api.py plans
+
+    -  `DATACENTER`:
+        This specifies the Linode datacenter where the Cluster Manager Linode is created. Set it to the ID of the datacenter that is nearest to your location, to reduce network latency. It's also recommended to create the cluster manager node in the same datacenter where the images and cluster nodes will be created, so that it can communicate with them using low latency private IP addresses and reduce data transfer usage.
+
+        To view the list of datacenters and their IDs:
+
+            source ~/storm-linode/api_env_linode.conf
+            ~/storm-linode/linode_api.py datacenters table
+
+    -  `DISTRIBUTION`:
+        This is the ID of the distribution to install on the Cluster Manager Linode. This guide has been tested only on Ubuntu 14.04 or Debian 8; other distributions are not supported.
```

Comment:
@alexfornuto Yes, it can already print out available distributions and kernels by querying Linode API.

Syntax is same:
            source ~/storm-linode/api_env_linode.conf
            ~/storm-linode/linode_api.py distributions table

and 

            source ~/storm-linode/api_env_linode.conf
            ~/storm-linode/linode_api.py kernels table

Again, do let me know if I can go ahead with adding this.